environments.yaml.tmpl

```diff
@@ -1,10 +1,24 @@
 environments:
- default:
-   values:
-   - ../etc/base.yaml
-   - ../etc/production.yaml
-   - ../etc/production.yaml.gotmpl
-   - ../etc/secrets.yaml
+  default:
+    values:
+      - ../etc/base.yaml
```

Comment:
Do you think it would be good to move these files helmfile related files to a separate directory as well? Probably something like `helmfile_configs` to put all of it's configuration related files in it.

Reply:
I think this would make sense yes. Should we do this now? We have to make sure all the scripts in the /bin folder play nice with this change.

Reply:
It would be probably better in the long run make things more organized but we can do it later if it is easier.